e2e-tests/tests/upgrade-consistency/01-create-cluster.yaml

```diff
@@ -10,6 +10,7 @@ commands:
 
       get_cr \
         | yq eval '.spec.crVersion="2.6.0"' - \
-        | yq eval ".spec.image=\"${IMAGE_BASE}:main-ppg$PG_VER-postgres\"" - \
-        | yq eval ".spec.backups.pgbackrest.image=\"${IMAGE_BASE}:main-ppg$PG_VER-pgbackrest\"" - \
+        | yq eval ".spec.image=\"${IMAGE_BASE}:pg-operator-2.6.0-ppg$PG_VER-postgres\"" - \
```

Comment:
Can we add some var for crVersion not to update it in `spec.crVersion=` & every IMAGE but just set it after functions?